pkg/auth/handler/webapp/authflow_login.go

```diff
@@ -43,7 +42,7 @@ func ConfigureAuthflowLoginRoute(route httproute.Route) httproute.Route {
 }
 
 type AuthflowLoginEndpointsProvider interface {
-	SSOCallbackURL(alias string) *url.URL
+	SSOCallbackURL(alias string, isDemo bool) *url.URL
```

Comment:
introduce a option struct

```golang
type SSOCallbackURL struct {
  Alias string
  UseSharedCredentials bool
}
```

Reply:
Added https://github.com/authgear/authgear-server/pull/5196/commits/09fe860defb9e8ed8eb0eac9769608bbdbaa50e6 to separate it into two methods